orc8r/cloud/helm/orc8r/charts/certs/templates/controller-certificate.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+{{- if .Values.create }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ .Release.Name }}-controller-certificate
+spec:
+  secretName: {{ .Release.Name }}-controller-tls
+  commonName: "*.{{ required "domainName must be provided" .Values.domainName }}"
+  subject:
+    countries:
+      - US
+  duration: {{ .Values.controller.duration }}
+  renewBefore: {{ .Values.controller.renewBefore }}
+  issuerRef:
+{{- if .Values.controller.customIssuer }}
+    name: {{ .Values.controller.customIssuer }}
+{{- else }}
+    name: {{ .Release.Name }}-root-issuer
+{{- end }}
+  dnsNames:
+  - "*.{{ required "domainName must be provided" .Values.domainName }}"
+  - "*.nms.{{ required "domainName must be provided" .Values.domainName }}"
```

Comment:
Why are we having controller.crt and nms.crt each cover both controller and nms domains?

Reply:
Keep both for backwards compatibilities.